doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
 
-* `org.uberfire.gzip.enable`: Enables or disables Gzip compression on GzipFilter. Default value: `true`.
-* `org.kie.workbench.profile`: Selects the {CENTRAL} profile. Possible values are `FULL` or `PLANNER_AND_RULES`. A prefix `FULL_` will set the profile and hide the profile preferences from the admin preferences. Default value: `FULL`
-* `org.appformer.m2repo.url`: {CENTRAL} uses the default location of the Maven repository when looking for dependencies. It points to the Maven repository inside {CENTRAL}. For example, `\http://localhost:8080/business-central/maven2`. Set this property before starting up {CENTRAL}. Default value: File path to the inner `m2` repository.
+* `org.uberfire.gzip.enable`: Enables or disables Gzip compression on the  `GzipFilter` compression filter. Default value: `true`.
+* `org.kie.workbench.profile`: Selects the {CENTRAL} profile. Possible values are `FULL` or `PLANNER_AND_RULES`. A prefix `FULL_` sets the profile and hides the profile preferences from the administrator preferences. Default value: `FULL`
+* `org.appformer.m2repo.url`: {CENTRAL} uses the default location of the Maven repository when looking for dependencies. It directs to the Maven repository inside {CENTRAL}, for example, `\http://localhost:8080/business-central/maven2`. Set this property before starting {CENTRAL}. Default value: File path to the inner `m2` repository.
 * `appformer.ssh.keystore`: Defines the custom SSH keystore to be used with {CENTRAL} by specifying a class name. If the property is not available, the default SSH keystore is used.
-* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in {CENTRAL} `.security` folder.
-* `appformer.experimental.features`: Enables the Experimental Features Framework.
-* `org.kie.demo`: Enables external clone of a demo application from GitHub.
-* `org.kie.workbench.controller`: The URL is used to connect with the {CONTROLLER}. For example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
-* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver.
-* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!.
-* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
+COMMENT: Default value?
+* `appformer.experimental.features`: Enables the experimental features framework.
+COMMENT: Default value?
+* `org.kie.demo`: Enables an external clone of a demo application from GitHub.
```

Comment:
`org.kie.demo` - There is no default property.

https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/service/ExamplesService.java#L34

https://github.com/kiegroup/jbpm-wb/blob/master/jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseDemoProvisioningService.java#L78